recipes/infinity/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "infinity" %}
+{% set version = "1.4" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "dc4aa138d7e366fc00d2e741e32c78a0fecd16b74f8daeb3f7408b459668005c" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
```

Comment:
Do you want to try [`noarch: python`](https://github.com/conda-forge/conda-forge.github.io/blob/master/src/meta.rst#building-noarch-packages)?

Reply:
I can, sure - should I also drop out the `script:` block below?

Reply:
`script` block still needs to be there